src/wazuh_db/wdb_syscollector.c

```diff
@@ -585,7 +585,7 @@ int wdb_package_insert(wdb_t * wdb, const char * scan_id, const char * scan_time
     sqlite3_bind_text(stmt, 12, multiarch, -1, NULL);
     sqlite3_bind_text(stmt, 13, source, -1, NULL);
     sqlite3_bind_text(stmt, 14, description, -1, NULL);
-    sqlite3_bind_text(stmt, 15, location, -1, NULL);
+    sqlite3_bind_text(stmt, 15, NULL != location ? location : "", -1, NULL); // Avoid bind NULL for agents older than 4.5.2
```

Comment:
"location" field in the multiple package wrappers has "UNKNOW_VALUE" (" ") as the default value.
Shouldn't be " " value be set here also as default?

Reply:
Ditto